components/ipfs/keys/ipns_keys_manager.cc

```diff
@@ -211,15 +216,15 @@ void IpnsKeysManager::UploadData(
 
   auto generate_endpoint = server_endpoint_.Resolve(kAPIKeyImportEndpoint);
   GURL url = net::AppendQueryParameter(generate_endpoint, kArgQueryParam, name);
-
   auto url_loader = CreateURLLoader(url, "POST", std::move(request));
+
   auto iter = url_loaders_.insert(url_loaders_.begin(), std::move(url_loader));
   iter->get()->DownloadToStringOfUnboundedSizeUntilCrashAndDie(
-      url_loader_factory_, base::BindOnce(&IpnsKeysManager::OnKeyImported,
-                                          weak_factory_.GetWeakPtr(), iter,
-                                          std::move(callback), name));
+      url_loader_factory_.get(),
```

Comment:
Why ApiRequestHelper isn't used here?

Reply:
this is request with custom HttpRequest, it will be addressed later